fusedrug/data/tokenizer/modulartokenizer/modular_tokenizer.py

```diff
@@ -1046,6 +1075,13 @@ def encode_list(
                     f"both padding token and padding id are None, but padding length is {max_len}. It's possible that it was set for truncation alone."
                 )
 
+        # Check if we have any unknown tokens in our input
+        if self.count_unknowns(merged_encoding) > 0:
+            if on_unknown == "raise":
+                raise Exception("Encountered unknown tokens in the input")
```

Comment:
I suggest to prepare a more detailed error / warning msg. 
The number of `<UNK>` or the input itself would be helpful

Reply:
The trouble is that our input is a list of TypedInput, and to fully parse it just for the exception would be a bit too much for such a small function. As a compromise, I changed this to only output the input string of first element of the list. This should identify the input sufficiently in most cases.